src/js/base/ScrollList.tsx

```diff
@@ -103,10 +118,12 @@ export const LegacyScrollList = ({ page, documents, pageCount, onLoadNextPage, r
     }
 
     return (
-        <StyledScrollList>
-            {entries}
-            {loading}
-        </StyledScrollList>
+        <ThemeProvider theme={theme}>
```

Comment:
Were you having problems without `ThemeProvider`? You shouldn't need it here

Reply:
Yeah since I added the border class, but with the styled method you sent it doesn't give this error anymore. Removed the `ThemeProvider` in the latest push